v21.2/cost-based-optimizer.md

```diff
@@ -265,10 +265,10 @@ SELECT * FROM abc@{NO_ZIGZAG_JOIN};
 ## See also
 
 - [`JOIN` expressions](joins.html)
-- [`SET (session variable)`](set-vars.html)
+- [`SET &lt;session variables&gt;`](set-vars.html)
```

Comment:
Further: in either case, the PR should be internally consistent with regard to where these code ticks are applied. I didn't call out lines like this because they appear more correct to me.

Reply:
Ah you're right there are 3 of these; I'll fix. Fuzzy brain all week. Will move everything to monospace and try my hand at left-nav. Reviews like this are why style guide updates are usually paired with concomitant correctness passes. Your time && mine.